18.04/php7.2/Dockerfile

```diff
@@ -161,7 +157,7 @@ RUN rm -rf /var/www/html \
 COPY files/apache-default-host.conf /etc/apache2/sites-available/000-default.conf
 COPY files/apache-default-host.conf /etc/apache2/sites-enabled/000-default.conf
 COPY files/apache-default-varnish-host.conf /etc/apache2/sites-available/000-default-varnish.conf
-COPY files/listen_8080.conf /etc/apache2/conf-available/listen_8080.conf
+#COPY files/listen_8080.conf /etc/apache2/conf-available/listen_8080.conf
```

Comment:
We do need this actually. It just never got added to the files dir properly during the migration to this image builder apparently.

See https://github.com/ProboCI/probo/blob/master/lib/plugins/TaskRunner/LAMPApp.js#L291 for where it is called in the LAMPApp plugin and https://github.com/ProboCI/probo-image-builder/blob/master/ansible/roles/ubuntu-apache/files/listen_8080.conf for the file contents.